runtime/gc_base/StringTable.cpp

```diff
@@ -356,7 +349,6 @@ static UDATA
 stringHashEqualFn(void *leftKey, void *rightKey, void *userData)
```

Comment:
This seems to return only TRUE or FALSE. I'd change the type to BOOLEAN here too (and it's inter-module wrapper, too), and stop there... There seem to be some core VM callers that treat the return value as UDATA. Upcasting from BOOLEAN to UDATA there should not be a problem - we are anyway doing it here and I what I want is just to move that upcast from this to higher up in call sequence, so that code on GC side is consistent.